HLTrigger/JetMET/src/HLTJetEtaTopologyFilter.cc

```diff
@@ -0,0 +1,103 @@
+/** \class HLTJetEtaTopologyFilter
+ *
+ *
+ *  \author Tomasz Fruboes
+ *     based on HLTDiJetAveFilter
+ */
+
+#include "HLTrigger/JetMET/interface/HLTJetEtaTopologyFilter.h"
+
+#include "DataFormats/Common/interface/Ref.h"
+#include "DataFormats/Common/interface/Handle.h"
+#include "DataFormats/Math/interface/deltaPhi.h"
+
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+
+#include<typeinfo>
+
+//
+// constructors and destructor
+//
+template<typename T>
+HLTJetEtaTopologyFilter<T>::HLTJetEtaTopologyFilter(const edm::ParameterSet& iConfig) : HLTFilter(iConfig),
+    inputJetTag_ (iConfig.template getParameter< edm::InputTag > ("inputJetTag")),
+    minPtJet_    (iConfig.template getParameter<double> ("minPtJet")),
+    //minPtJet3_   (iConfig.template getParameter<double> ("minPtJet3")),
+    jetEtaMin_     (iConfig.template getParameter<double> ("minJetEta")),
+    jetEtaMax_     (iConfig.template getParameter<double> ("maxJetEta")),
+    applyAbsToJet_ (iConfig.template getParameter<bool> ("applyAbsToJet")),
+
+    triggerType_ (iConfig.template getParameter<int> ("triggerType"))
+{
+    m_theJetToken = consumes<std::vector<T>>(inputJetTag_);
+    LogDebug("") << "HLTJetEtaTopologyFilter: Input/minDphi/triggerType : "
+        << inputJetTag_.encode() << " "
+        << triggerType_;
+}
+
+template<typename T>
+HLTJetEtaTopologyFilter<T>::~HLTJetEtaTopologyFilter(){}
+
+template<typename T>
+void
+HLTJetEtaTopologyFilter<T>::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+    edm::ParameterSetDescription desc;
+    makeHLTFilterDescription(desc);
+    desc.add<edm::InputTag>("inputJetTag",edm::InputTag("hltIterativeCone5CaloJets"));
+    desc.add<double>("minPtJet",50.0);
+    //desc.add<double>("minPtJet3",99999.0);
+    desc.add<double>("minJetEta", -1.);
+    desc.add<double>("maxJetEta", 1.4);
+    desc.add<bool> ("applyAbsToJet", false),
+    desc.add<int>("triggerType",trigger::TriggerJet);
+    descriptions.add(std::string("hlt")+std::string(typeid(HLTJetEtaTopologyFilter<T>).name()),desc);
```

Comment:
The synatx here has changed for 74X, it must be like
```
descriptions.add(defaultModuleLabel<HLTAlphaTFilter<T>>(), desc);
```
of course with the ciorrect class name.